pages/1.12/deploying-services/fault-domain-awareness/index.md

```diff
@@ -79,7 +79,7 @@ User-created Marathon services and pods support both zone and region awareness.
 
 ## Marathon services and pods
 
-In your Marathon service or pod definition, you can use [placement constraints](/1.11/deploying-services/marathon-constraints/) to:
+In your Marathon service or pod definition, you can use [placement constraints](/1.12/deploying-services/marathon-constraints/) to:
```

Comment:
should there be mention of Metronome as well ... this page is generically "fault-domain-awareness" regardless of framework